src/Illuminate/Auth/Middleware/Authenticate.php

```diff
@@ -0,0 +1,70 @@
+<?php
+
+namespace Illuminate\Foundation\Http;
+
+use Closure;
+use Illuminate\Auth\AuthenticationException;
```

Comment:
once you fix the namespace, styleci should actually remove this import automatically for us ;)